source/draw/gpu/opengl/backend/GLExtensions.ooc

```diff
@@ -29,8 +29,13 @@ GLExtensions: class {
 			This eglClientWaitSyncKHR = This _load("eglClientWaitSyncKHR") as Func (Pointer, Pointer, Int, ULong) -> Bool
 			version(!windows) {
 				//For some reason this function can't be loaded with eglGetProcAddress so we load it with dlsym instead
-				handle := null //This should be RTLD_DEFAULT, defined in dlfcn.h
-				This eglDupNativeFenceFDANDROID = Closure fromPointer(dlsym(handle, "eglDupNativeFenceFDANDROID")) as Func(Pointer, Pointer) -> Int
+				//dlsym can also be called with the special flag RTLD_DEFAULT, defined in dlfcn.h
```

Comment:
Could probably remove this comment but I saved it for the moment as we might want to use it instead.